src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,349 @@
+package scala                     //TODO: Change this strawman --> we just need sizeHintIfCheap from scala
```

Comment:
It was meant to convey whether or not you already know the size, not whether it is knowable / finite.